src/network/networkd-address-generation.c

```diff
@@ -252,8 +252,8 @@ static int generate_addresses(
         return 0;
 }
 
-int dhcp_pd_generate_addresses(Link *link, const struct in6_addr *prefix, uint8_t prefixlen, Set **ret) {
-        return generate_addresses(link, link->network->dhcp_pd_tokens, &DHCP_PD_APP_ID, prefix, prefixlen, ret);
+int dhcp_pd_generate_addresses(Link *link, const struct in6_addr *prefix, Set **ret) {
+        return generate_addresses(link, link->network->dhcp_pd_tokens, &DHCP_PD_APP_ID, prefix, 64, ret);
```

Comment:
`64` is used twice, maybe add a `#define` for it?

Reply:
Is it? I do not think so... At most, I'd like to leave this to another PR.

Reply:
I don't think we want to do this, because "/64" is used in many places in strings, and we can't use a define there. (We could do something more complicated with templating, but that seems vastly overkill.)  I'd say that "64" is a well-known-constant and we don't need a define for it.